bigbone/src/test/kotlin/social/bigbone/api/method/DomainBlockMethodsTest.kt

```diff
@@ -29,7 +29,7 @@ class DomainBlockMethodsTest {
 
         verify {
             client.get(
-                path = "/api/v1/domain_blocks",
+                path = "api/v1/domain_blocks",
```

Comment:
Could we also introduce the `private val endpoint`constant for test classes?. I had a quick look and there are lots of places where we have duplicate endpoint strings.

Reply:
I didn’t want to change that here and think we should first discuss in #396 which way to go with this. There are multiple options, some of which might be:
1. Using `internal` visibility for the endpoint `val`s
2. Introducing an endpoint `val` per test, like we do in the methods class under test
3. Having a central location for _all_ endpoints
4. …?

Reply:
Ah ok, I did not check https://github.com/andregasser/bigbone/issues/396 yet. Fine for me.